Specs/Scene/PostProcessStageCompositeSpec.js

```diff
@@ -169,6 +169,7 @@ defineSuite([
 
     it('does not run a stage that requires depth textures when depth textures are not supported', function() {
         var s = createScene();
+        s.postProcessStages.fxaa.enabled = true;
```

Comment:
Should this test require FXAA to be on?  Are we sure it's not just masking an unrelated bug?

Reply:
Feel free to write up a new issue for this, I just don't want to assume everything is okay.

Reply:
@mramato I did some debugging but decided to ship the temporary fix until I know if what I observed was a consequence of our unit tests or if it was something of a larger issue. Until then I am confident that it is not related to any new code. 

I will take a look into it more soon to try and get a more exact understanding.

Reply:
That's fine but please write up an issue for it so we don't forget about it and people who know more than us can chime in with their thoughts.  Rule number 1, always write it down.

Reply:
@mramato yeah I have notes but wanted to take a second look before I post it as a GitHub  issue.